txbits/public/js/password-form.js

```diff
@@ -41,5 +41,21 @@ $(function(){
         }
         $password1.keyup(check_form);
         $password2.keyup(check_form);
+    }).submit(function(e) {
+        var $this = $(this);
+        var min_len = $this.attr('data-password-length');
```

Comment:
I know. I'm just saying that you copied and pasted the code above. You can wrap it in a function instead.